tests/Umbraco.Tests.UnitTests/Umbraco.Core/Cache/RuntimeAppCacheTests.cs

```diff
@@ -13,18 +13,23 @@ public abstract class RuntimeAppCacheTests : AppCacheTests
     internal abstract IAppPolicyCache AppPolicyCache { get; }
 
     [Test]
-    [Explicit("Testing for timeouts cannot work on VSTS.")]
+    // ToDo: Check wether the Explicit attribute is still needed e.g. in pipelines
```

Comment:
I think we'll still want the `Explicit` here as the test will be slow due to the artificial delay.

Reply:
Ok, I got the point, added the `Explicit` attribute back.
But I like to change the misleading message. What Do you think?

Reply:
Agreed, I've reduced this and removed the `Explicit` attribute.  Doesn't seem there is any issue running this locally or on the pipeline.